built-in-references/KubernetesService/pod-enforce-labels/limited-preview/azurepolicy.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "displayName": "[Limited Preview]: Enforce labels on pods in AKS",
+    "policyType": "BuiltIn",
+    "mode": "Microsoft.ContainerService.Data",
+    "description": "This policy enforces the specified labels are provided for pods in an Azure Kubernetes Service cluster. Limited Preview policies only work for registered subscriptions. To register, please go to https://aka.ms/akspolicyonboarding. For instruction on using this policy, please go to https://aka.ms/akspolicydoc.",
+    "metadata": {
+      "category": "Kubernetes service"
+    },
+    "parameters": {
+      "commaSeparatedListOfLabels": {
```

Comment:
>commaSeparatedListOfLabels" [](start = 7, length = 27)

are all of these required on every pod or does each pod need at least one?  Its not clear from the policy or param description

Reply:
All the labels are required, I will update the description.

Reply:
FYI, These definitions are in ARM repo as well. I am planning to add data manifests to ARM repo, will update this description in ARM repo when I add data manifests.